pages/services/ember-consulting.vue

```diff
@@ -61,10 +61,10 @@
 
     <TwoColumnSection
       class-string="mt-20"
+      header="We’ll never leave you high and dry"
       :image="marketing"
       :is-flipped="true"
     >
-      <template #header>We’ll never leave you high and dry</template>
```

Comment:
I think I would suggested keeping this slots like this maybe. What do you think?

Reply:
I changed it to accommodate when it's just a text box and image without a header - otherwise there's still an empty header in the page. I could change it so that the slot takes both header & paragraph copy though?
![Screen Shot 2022-03-07 at 3 15 47 PM](https://user-images.githubusercontent.com/62962173/157134244-9216d37d-6f5b-4f58-8117-837a0e7ca7b5.png)